annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/method/MethodRegistry.java

```diff
@@ -94,7 +96,7 @@ protected Builder<MethodRecord> getBuilder() {
       node.fillInformation(
           new OnMethod(Helper.deserializePath(values[8]), values[1], values[2]),
           parentId,
-          Boolean.parseBoolean(values[5]),
+          values[5].split(ANNOTATION_DELIMITER),
```

Comment:
Presumably we need to bump the serialization version?  Do we need to worry about backwards compat?

Reply:
Sorry, I realize this comment is invalid as we are only changing the Scanner output, not NullAway's output.

Reply:
I believe we don't promise compatibility between anything but the exact same version of `annotator-scanner` and `annotator-core` 😉